### DISCUSSION TITLE
Users migration from Wordpress to Firebase Auth

### DISCUSSION THREAD

**Discussion by gabyshev** [dalenguyen/firebase-wordpress-plugin#156] [Q&A]
Hi!

I see that the plugin provides a lot of features. We are planning to discontinue using WordPress and looking for ways to seamlessly move users to a new platform without resetting their password.

Is it possible to move existing users from WordPress to Firebase Auth so that they can continue using the same passwords?

**Comment by dalenguyen** [comment]
Yes, you can take a look at this add-on. 

https://firebase-wordpress-docs.readthedocs.io/en/latest/extensions/firebase-users.html 

How many users do you have in WordPress?

**Reply by gabyshev** [reply]
less than 10 000

Is there any limit in the number of users?
